web/index.html

```diff
@@ -376,32 +437,36 @@ <h3>Used in</h3>
             return queryParams;
         }
 
-        // Decode base64 value mod 1000
-        function getChunk(value) {
-            let base64 = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-='
-            let result = 0
-            for (let i = 0; i < value.length; i++) {
-                result = (result * 64 + base64.indexOf(value[i]))
-            }
-            return Math.floor(result / 100)
-        }
-
-        const DATA_PREFIX = 'data/';
+        const PATH_PREFIX = window.location.host == 'expitau.github.io' ? '/' : '';
+        const CHUNK_WORKER_PATH = `${PATH_PREFIX}loadChunkWorker.js`;
+        const DATA_PATH = `${PATH_PREFIX}data/`;
+        const HELPER_PATH = window.location.host == 'expitau.github.io' ? 'https://expitau.github.io/InfiniteCraftWiki/helper'
+            : 'http://127.0.0.1:5501/web/helper.html';
 
         // Vue app
         const app = new Vue({
             el: '#app',
             data: {
                 data: { index: {}, list: [], costs: {}, craftingCache: {}, data: {}, metadata: { recipeCount: 0, loadedChunks: [] } },
+                chunkLoadState: [],
+                craftingTreeData: null,
+                bookmarkletData: "",
                 currentElement: "",
+                elementLoaded: false,
                 showHiddenRecipes: false,
                 searchQuery: "",
                 showToast: false,
-                saveData: [],
+                showImportToast: false,
+                isDraggingBookmarklet: false,
+                isShaking: false,
+                saveData: new Set(),
+                saveDataArray: [],
                 loaded: 1000,
                 loadedElements: 100,
                 currentPanelIndex: 0,
                 scroll: 0,
+                isScrolling: false,
+                showFullRecipe: false,
```

Comment:
This is used as the toggle to tell if we should display the full recipe or use the users elements to simplify the recipe